src/screens/Home/HomeScreen.tsx

```diff
@@ -1,29 +1,75 @@
 import * as React from 'react';
-import { StyleSheet, View, TouchableOpacity, Text, Pressable} from 'react-native';
+import { StyleSheet, View, TouchableOpacity, Text, Pressable, Button, Modal, Image} from 'react-native';
 import EditScreenInfo from '../../components/EditScreenInfo';
 import { getAllTransactions } from '../../firebase/firestore/transaction';
 import { useEffect } from 'react';
 import ViewContainer from '../../components/ViewContainer';
 import { Divider, Title } from 'react-native-paper';
 import styles from './styles';
 import globalStyles from "../../globalStyles";
+import { ProgressSteps, ProgressStep } from 'react-native-progress-steps';
+import {moment} from 'moment';
+import StepIndicator from 'react-native-step-indicator';
+
+const labels = ["1000", '2000', '3000'];
+
+const getCurrentDate=()=>{
+    var moment = require('moment');
+    return (moment().format('dddd[,] MMMM Do')); 
+}
+
+const customStyles = {
```

Comment:
Please import this from the styles page! you may need to do another export (try export const customStyles)